onboarding/src/Components/hooks/Packages.js

```diff
@@ -29,7 +29,7 @@ function Packages(props) {
           isLoaded(true);
           setRows(result);
           const ids = result.map((res) => res.id);
-          const maxId = ids.reduce((a, b) => Math.max(a, b));
+          const maxId = ids.reduce((a, b) => Math.max(a, b));/* Uncaught (in promise) TypeError: reduce of empty array with no initial value; */
```

Comment:
Why not `const maxId = Math.max(...ids);`? ;)

Reply:
I experienced this `typeError` when I didn't have any package or page.

Reply:
Yes, me too, so I propose a little code change like above.

Reply:
Any small check if `ids` is not empty should be fine :)

Reply:
Of course, I just think that reduce method is unnecessary here.